features/comment_types.feature

```diff
@@ -1,16 +1,27 @@
-@wip
 Feature: Users can choose a type of comment on app
 
 	AS a staff that manages the matching between a project and customers
 	SO THAT I can easily keep track of an app's progress of customer matching
 	I WANT the app page have different comment types not only for general comments, but also for contact progress
 
 Background:
+	And the following users exist:
+		| name 				| github_uid 		| email 						|
```

Comment:
Very small comment but you could align all comments so that it is easier to skim over the file. Same applies for "Scenario: User can group comments by type". Very extensive and concise tests overall!